internal/command/views/operation.go

```diff
@@ -95,6 +95,12 @@ func (v *OperationHuman) EmergencyDumpState(stateFile *statefile.File, enc encry
 }
 
 func (v *OperationHuman) Plan(plan *plans.Plan, schemas *tofu.Schemas) {
+	// If the -show-sensitive argument is provided in the tofu plan/apply command,
+	// then mark the sensitive variable in plan.Changes.Outputs as false.
+	if v.view.showSensitive && plan.Changes != nil {
+		resetSensitiveVariables(plan.Changes.Outputs)
```

Comment:
I read the issue, and I'm unsure if we should only show sensitive outputs when it comes to `plan` / `apply` or the whole plan.
From the [issue](https://github.com/opentofu/opentofu/issues/839):
> There are multiple cases where TF knows a sensitive value but refuses to display it, such as:
1. viewing TF created secret values
2. approving sensitive diffs


From reading this, I understand that all sensitive changes should be seen, not only outputs, unless I misunderstood.